Cargo.toml

```diff
@@ -117,15 +117,19 @@ codecov = { repository = "clap-rs/clap-verbosity-flag" }
 default = ["log"]
 log = ["dep:log"]
 tracing = ["dep:tracing-core"]
+serde = ["dep:serde"]
 
 [dependencies]
 clap = { version = "4.0.0", default-features = false, features = ["std", "derive"] }
 log = { version = "0.4.1", optional = true }
+serde = { version = "1", features = ["derive"], optional = true }
```

Comment:
Is the `derive` worth it?  As this is a library meant for CLIs and they are already using `clap` with `derive` (which I would consider making optional) and the rest of the CLI is already likely using `serde`, this is probably fine.